ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -157,6 +157,17 @@ FCU_DFSS_DSP_HP2ZC:
   - DSP
   - HP2ZC
 
+FCU_DFSS_DFMSC_DSPRTC_HPDSPRTC:
```

Comment:
We don't need `DSPRTC` if we have `HPDSPRTC`.

Reply:
There is a similar type that I used as a base to create this one (FCU_DFSS_DSPRTC_DXDSPRTC), wanted to keep it consistent. Should I still remove DSPRTC?